docs/UserGuide.adoc

```diff
@@ -105,11 +105,34 @@ Edits the name of the 2nd staff to be `Betsy Crower` and clears all existing tag
 Rates an existing staff in SSENISUB +
 Format: `rate INDEX r/8`
 
+****
+* Rates the staff at the specified `INDEX`. The index refers to the index number shown in the displayed staff list. The index *must be a positive integer* 1, 2, 3, ...
+* The rating field must be provided.
+* Ratings can only be from 1 - 10.
+* Existing values will be updated to the input values if they are different.
+****
+
 Examples:
 
 * `rate 3 r/8` +
 Rate the 3rd staff of a rating 8.
 
+=== Rating a staff: `feedback`
```

Comment:
Change it to giving feedback to a staff? might confuse others with rating as giving the rate for the staff